index.js

```diff
@@ -262,6 +263,9 @@ class Autohook extends EventEmitter {
     
     if (!webhookUrl) {
       this.startServer();
+      if(this.auth.ngrok_secret){
+        await ngrok.authtoken(token)
+      }
```

Comment:
Style: indent and spacing. Add a space between `if` and its bracket; add a space between the `if` ending bracket and the square bracket; add a ; at the end of the statement:

```js
if (this.ngrokSecret) {
  await ngrok.authtoken(token);
}